lambo

```diff
@@ -319,6 +333,14 @@ if [[ "$AUTH" = true ]]; then
     php artisan make:auth
 fi
 
+if [[ "$DB_USERNAME" != "" ]]; then
```

Comment:
What are these lines for?

Reply:
TBH ... not sure why I put them there. :see_no_evil: